src/hb-coretext.cc

```diff
@@ -62,10 +63,32 @@ reference_table  (hb_face_t *face HB_UNUSED, hb_tag_t tag, void *user_data)
 			 release_table_data);
 }
 
+static void
+hb_face_from_ct_font_destroyer(void *user_data)
+{
+}
```

Comment:
Just want to explain the cause and the fix, I was not doing "retain" somewhere else so I could free these on a different place but after your mention I put a retain on the custom constructors and trigged a ct_font "release" here.